scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -14,6 +14,8 @@ public class CompileTargets
     {
         public static readonly string CoreCLRVersion = "1.0.1-rc2-23811";
         public static readonly string AppDepSdkVersion = "1.0.6-prerelease-00003";
+        public static readonly bool IsWinx86 = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) &&
```

Comment:
Understand why you did this, but after #1504 goes in, this could ideally use CurrentPlatform and CurrentArchitecture